### ISSUE TITLE
Change font of text in-game, and maybe position 

### ISSUE THREAD

**Issue by joshhartford** [api-Hypernova/AC2-cube2#55]
The text at the bottom left of the screen that informs you about what you picked up and etc.. should be changed to fit the theme of the game. Maybe through Sausage Fattener on it and squish it out? LOL just make it more retro-goofy-weeb like. Also, perhaps move the body of text to the bottom middle of the screen instead of on the left? I think it will need some trial and error.

**Comment by api-Hypernova** [comment]
Yeah I wanted to fix this for a long time. Basically there are 2 ways to display text on the screen - printing to the console or chat log (which stacks up and slowly fades away) or you can actually put a blob of text anywhere in the screen at some coordinates, but it won't fade or anything automatically. For the pickups we should definitely put the text in the center of the screen to give more attention (like in UT).
